src/components/tooltip/Marker.js

```diff
@@ -693,11 +701,13 @@ export const TooltipMarker = explicitConnect<OwnProps, StateProps, {||}>({
     const thread = selectors.getThread(state);
     const implementationFilter = getImplementationFilter(state);
     const pages = getPageList(state);
+    const zeroAt = getZeroAt(state);
     return {
       threadName,
       thread,
       implementationFilter,
       pages,
+      zeroAt,
```

Comment:
You'll see below that we use it to display the position of the preconnect phase.

The good thing is that we can also use it later to display the start time of any marker.